data/vtlib/bwa_aln_alignment.json

```diff
@@ -81,11 +81,10 @@
 		"use_STDIN":true,
 		"use_STDOUT":true,
                 "cmd":[
-			"scramble",
-			{"subst":"s2b_compress_level", "ifnull":"-0"},
-			{"subst":"s2b_mt", "ifnull":{"subst_constructor":{ "vals":[ "-t", {"subst":"s2b_mt_val"} ]}}},
-			"-I", "sam",
-			"-O", "bam"
+			{"subst":"samtools_executable"}, "view", "-b",
+			{"subst_constructor":{"vals":[ "--output-fmt-option", {"subst_constructor":{"vals":[ "level", {"subst":"s2b_compression_level","ifnull":0} ], "postproc":{"op":"concat", "pad":"="}}}]}},
```

Comment:
If I'm reading this correctly the --output-fmt-option value appears to be the compression level